src/models/user.ts

```diff
@@ -43,6 +44,9 @@ export const UserModel = types
       self.className = user.className;
       self.classHash = user.classHash;
       self.offeringId = user.offeringId;
+      if (user.firebaseJWT && (user.firebaseJWT as PortalFirebaseStudentJWT).returnUrl) {
+        self.loggingRemoteEndpoint = (user.firebaseJWT as PortalFirebaseStudentJWT).returnUrl;
```

Comment:
I'll take your word for it that `returnUrl` corresponds to the `run_remote_endpoint`.